qiskit/dagcircuit/__init__.py

```diff
@@ -17,8 +17,8 @@
 
 .. currentmodule:: qiskit.dagcircuit
 
-DAG Circuits
-============
+DAG (directed acyclic graph) Circuits
+=====================================
```

Comment:
It's unusual to put the definition of an acronym in a title.  Probably a bit better to say it as:

> DAG Circuits
>
> Quantum operations in the absence of control-flow operations can be represented by a directed acyclic graph (DAG).
> ...

or have the title as "Circuits as Directed Acyclic Graphs" and define the "DAG" acronym in the prose.

Reply:
done in 430d0b3f709fa10879133e76f0fc75f55df9fe9e